app/models/customer.rb

```diff
@@ -1,3 +1,4 @@
 class Customer < ApplicationRecord
   validates_presence_of :first_name, :last_name
+  has_many :invoices
```

Comment:
Should we add 'has_many :items, through :invoices' and other secondary relationships here? Or can we get those relationships through joins queries?

Reply:
I think the best way to do it is to use invoice_items as a joins table and implement has_many through relationships. I think it probably will simplify the way we might need to structure queries and use active record.

Reply:
Good call. I will make some changes and get this updated.